security/fma-superchainerc20.md

```diff
@@ -0,0 +1,101 @@
+
+### FM1: Unauthorized Access to `crosschainMint` & `crosschainBurn` Functions
+
+- **Description:** The `crosschainMint` and `crosschainBurn` functions can only be called by the `SuperchainTokenBridge`,  enforced by the check `msg.sender != Predeploys.SUPERCHAIN_TOKEN_BRIDGE`. If the bridge address is badly defined or the modifier bypassed, an entity could mint and burn tokens.
+- **Risk Assessment**: Medium.
+    - Potential impact: High. All tokens based on this implementation could be potentially at risk.
+    - Likelihood: Very Low. `Predeploys.SUPERCHAIN_TOKEN_BRIDGE` is defined via protocol upgrades. The conditional is sufficiently simple and battle-tested to give confidence in the implementation.
+- **Mitigation**: Current unit tests perform authorization checks for `crosschainMint` ([test](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/test/L2/SuperchainERC20.t.sol#L38)) and `crosschainBurn` ([test](https://github.com/ethereum-optimism/optimism/blob/1add88a41b8cdc5c488a18976e3e4fa4b02e7da4/packages/contracts-bedrock/test/L2/SuperchainERC20.t.sol#L77)). There should be tests to ensure that the `SuperchainTokenBridge` predeploy is correctly set during the interop fork upgrade, and is not subject to unexpected changes.
+- **Detection**: Ideally, run an off-chain agent to monitor every `crosschainMint` and `crosschainBurn` event in real-time, perhaps it could be resource-intensive given the permissionless usage of `SuperchainERC20` and the expected large number of deployments. If that’s not feasible, rely on user-filed support tickets to flag unusual mint/burn activity, which the security team can investigate for suspected unauthorized access.
+- **Recovery Path(s)**: Equivocation (i.e. another implementation being set) on the `SuperchainTokenBridge` address would require a protocol upgrade or hard fork to restore the expected code.
+
+### FM2: Token contract not Deployed in Destination Chain, but Able to Deploy
+
+- **Description**: When a cross-chain transfer is initiated (via `sendERC20`), the destination chain must have the  `SuperchainERC20` contract deployed at the same address. If the contract does not exist on the destination chain, the `relayERC20`  cannot mint the tokens and thus will fail.
+- **Risk Assessment**: Medium
+    - Potential impact: Medium. Users can lose access to their tokens temporarily or permanently if the deployer (in the case of permissioned tokens) doesn’t deploy the token afterwards.
+    - Likelihood: Medium. A deployer may choose not to deploy the token on all chains.
+- **Mitigation**: Trusted bridge interfaces should prevent users from sending tokens to a chain where the token doesn’t exist. Double check other trusted sources (such as Superchain Token List) for greater confidence.
+- **Detection**: Support tickets filed by users reporting the issue.
+- **Recovery Path(s)**: Deploy the token.
+
+### FM3: Token Deployer is Lost, or Unable to Deploy to Expected Address
```

Comment:
Clarify this is a variant of FM2 where you can't deploy the token on the other chain therefore funds are lost